src/components/Misc/index.ts

```diff
@@ -0,0 +1 @@
+export * from './RsvProgressBar';
```

Comment:
Immer wenn ich "Misc" (und ähnlich) lese, gehen meine Warnglocken an. 

Ich bin [in Safetycheck](https://github.com/FixMyBerlin/fixmy.safetycheck/tree/develop/src/components) dazu übergegangen, in Components einen Ordner pro "page" anzulegen und dort dann die Components einzusortieren, die nur in der Page verwendet werden.

Dieser hier gehört ja sicher zu einer konkreten Page-Strutkur(?). Und vermutlich könnte man dann die RsvDetails auch dort hin legen. Im Layouts-Ordner sollte IMO nur die Wrapper-Layouts liegen und so Dinge wie Navi, Footer.

Reply:
Klingt gut. Wuerde die umstrukturierung in ne extra PR machen, nachdem ich diese und #47 in develop gemerged habe.